js/plot/cbg.js

```diff
@@ -29,9 +30,9 @@ module.exports = function(pool, opts) {
   var defaults = {
     bgUnits: 'mg/dL',
     classes: {
-      low: {boundary: 80},
-      target: {boundary: 180},
-      high: {boundary: 200}
+      low: { boundary: 70 },
+      target: { boundary: 180 },
+      high: { boundary: 300 },
```

Comment:
On the next pass I think we might want to delete all these defaults (as we should *always* be passing a set of categories/classes in) and replace with throwing an error at least in development so that we can be sure when folks customize their range it actually appears customized everywhere. Open to other ideas of how to accomplish this...